spec.bs

```diff
@@ -934,6 +983,30 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
       1. Set |auctionConfig|'s [=auction config/seller signals=] to the result of
         [=serializing a JavaScript value to a JSON string=], given |result|.
     * To handle an error, set |auctionConfig|'s [=auction config/seller signals=] to failure.
+1. If |config|["{{AuctionAdConfig/auctionNonce}}"] [=map/exists=]:
+  1. Set |auctionConfig|'s [=auction config/auction nonce=] to the result of running
+    [=get uuid from string=] with |config|["{{AuctionAdConfig/auctionNonce}}"].
+1. If |config|["{{AuctionAdConfig/additionalBids}}"] [=map/exists=]:
+  1. [=exception/Throw=] a {{TypeError}} if any of the following conditions hold:

```

Comment:
Every time I review a PR in this algorithm, I have to ask myself whether or not this validation algorithm is run in parallel or not (aka whether we can actually `[=throw=]` and do main-thread-ey things). Can you file a follow-up (or just quickly make a follow-up PR) to make this algorithm's first step clearly `[=Assert=]` that it is *not* running in parallel?

Reply:
Done. #848